suite-native/module-send/src/components/CoinControl/UtxoCard.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+    AccountAddressFormatter,
+    CryptoAmountFormatter,
+    FiatAmountFormatter,
+} from '@suite-native/formatters';
+import {
+    RootStackParamList,
+    RootStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Utxo } from '@trezor/blockchain-link-types';
+
+export type Props = {
+    utxo: Utxo;
+    onToggle: (utxo: Utxo) => void;
+    accountKey: string;
+    isSelected?: boolean;
+};
+
+type TransactionDetailNavigation = StackNavigationProps<
+    RootStackParamList,
+    RootStackRoutes.TransactionDetail
+>;
+
+export const UtxoCard = ({ utxo, onToggle, accountKey, isSelected = false }: Props) => {
+    const { DateFormatter } = useFormatters();
+    const navigation = useNavigation<TransactionDetailNavigation>();
+
+    const transactionBlockTime = useSelector((state: TransactionsRootState) =>
+        selectTransactionBlockTimeById(state, accountKey, utxo.txid),
+    );
+
+    const fiatCurrencyCode = useSelector(selectLocalCurrency);
+    const fiatRateKey = getFiatRateKey('btc', fiatCurrencyCode);
+
+    const currentRates = useSelector((state: FiatRatesRootState) =>
+        selectFiatRatesByFiatRateKey(state, fiatRateKey),
+    );
+
+    const handleShowDetails = () => {
+        navigation.push(RootStackRoutes.TransactionDetail, {
+            txid: utxo.txid,
+            accountKey,
+        });
+    };
+
+    const fiatTotalActualNumeric = pipe(
+        convertCryptoToFiatAmount({
+            amount: utxo.amount,
+            symbol: 'btc',
+            rate: currentRates?.rate,
+        }),
+        Number,
+    );
+
+    return (
+        <Card noPadding borderColor={isSelected ? 'backgroundSecondaryDefault' : 'transparent'}>
+            <VStack>
+                <HStack
+                    paddingTop="sp16"
+                    paddingHorizontal="sp12"
+                    justifyContent="space-between"
+                    alignItems="center"
+                >
+                    <VStack>
+                        <HStack alignItems="center">
+                            <CryptoAmountFormatter
+                                color="textDefault"
+                                variant="highlight"
+                                value={utxo.amount}
+                                isBalance={false}
+                                symbol="btc"
+                            />
+                            <Text color="textSubdued">≈ </Text>
+                            <FiatAmountFormatter
+                                color="textSubdued"
+                                variant="highlight"
+                                symbol="btc"
+                                value={String(fiatTotalActualNumeric)}
+                            />
+                        </HStack>
+
+                        <AccountAddressFormatter
+                            style={{
+                                maxWidth: '80%',
+                            }}
```

Comment:
no inline styles please
```suggestion
                            style={applyStyle(...)}
```

Reply:
ca79ab1df38447a6c4cfe999a927ae41a793df59